consul-lambda/consul-lambda-registrator/upsert_event.go

```diff
@@ -118,6 +118,7 @@ func (env Environment) storeServiceDefaults(e UpsertEvent) error {
 }
 
 func (env Environment) upsertTLSData(e UpsertEvent) error {
+	var advancedTier bool
```

Comment:
I think this needs to get removed because it will cause a compile error (unused variable):
```suggestion
```

Reply:
Good catch.  Fixed